libglusterfs/src/inode.c

```diff
@@ -523,9 +522,9 @@ __inode_unref(inode_t *inode, bool clear)
 
     --inode->ref;
 
-    index = __inode_get_xl_index(inode, this);
+    index = __inode_get_xl_index(inode, xl);
     if (index >= 0) {
-        inode->_ctx[index].xl_key = this;
+        inode->_ctx[index].xl_key = xl;
```

Comment:
nit (and in a different patch) - I don't see why this line is needed at all.

Reply:
Noted. `xl_key` is already set to `xl` when `index > 0`, so we can safely remove this line. Patch coming soon.

Reply:
Proposed at: #3207